src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -87,6 +87,11 @@ public function newAction(Request $request)
             $em->persist($post);
             $em->flush();
 
+            // Flash messages are used to notify the user about the result of the
+            // actions. They only last as long as the next request.
```

Comment:
We should improve this description like we did in the docs (flash messages last until they are retrieved from the bag).